src/main/scala/io/appthreat/atom/Atom.scala

```diff
@@ -351,8 +361,20 @@ object Atom:
                             AtomDataFlowSlice(x, DataFlowGraph.buildFromSlice(x).paths).toJson
                         )
                     saveSlice(config.outputSliceFile, atomDataFlowSliceJson)
-                case _: UsagesConfig =>
+                case u: UsagesConfig =>
                     saveSlice(config.outputSliceFile, sliceCpg(ag).map(_.toJson))
+                    if u.extractEndpoints then
+                        val result = ExternalCommand.run(
+                          s"atom-tools convert -i ${config.outputSliceFile} -t ${config.language} -f openapi3.1.0 -q -o ${config.inputPath.pathAsString}${java.io.File.separator}openapi.generated.json",
```

Comment:
This is the place where atom-tools is invoked.

Reply:
@prabhu Cool! I think I had still better create an atom-tools container and probably binaries though as fwdsec wish to utilize filtering - unless you want to allow that here.

Reply:
@cerrussell changed the dockerfile to use CMD so atom-tools can be invoked directly from atom container image too